bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/RemoteMethodType.java

```diff
@@ -27,4 +27,8 @@
     ObjectType getParentObjectType();
 
     FunctionType getType();
+
+    String[] getParamNames();
+
+    Boolean[] getParamDefaultability();
 }
```

Comment:
Shall we return a class with the ParamName and the defaultability status? This is not nice to have two functions to return the details about Paramters

Reply:
+1. But in that case, it's better to change the resource functions too. That'll be a kinda big change though.

Reply:
Since we need default values +1 to have that change in both resource and remote methods.
ex. 
Param[] getParameters();

Param can contain name and defaultability + default value if there is any